### ISSUE TITLE
Grafana Helm Chart completely broken (secrets checking)

### ISSUE THREAD

**Issue by rlees85** [grafana/helm-charts#2914]
New functionality seems to have completely crippled the chart.

Relating to this helper:
```
{{/*
    Checks whether the user is attempting to store secrets in plaintext
    in the grafana.ini configmap
*/}}
{{/* grafana.assertNoLeakedSecrets checks for sensitive keys in values */}}
{{- define "grafana.assertNoLeakedSecrets" -}}
      {{- $sensitiveKeysYaml := `
```

Can no longer mount secrets as files. Short example
```
  grafana.ini:
    database:
      password: "$__file{/_secrets/database/password}"
```

This completely bans all of Grafanas `$__` helpers and forces you to push everything through as environment variables.
Please revert or at least add an "I know what I am doing" flag. This is crippling

**Comment by rlees85** [comment]
Found the override:
```
################################################################################

assertNoLeakedSecrets: false

################################################################################
```

and it works thankfully

**Comment by bdalpe** [comment]
Also reported in #2896 and #2899. PR (#2904) is open for fix.

**Comment by bdalpe** [comment]
@rlees85 this was fixed with chart version 7.2.2.
